manual_flagging/yaml/inputs.yaml

```diff
@@ -13,7 +13,7 @@ run_note: |
 # without flags, it assigns a version number of "1". The latest
 # version of each sale flag can be found in the "default"."vw_pin_sale" view.
 
-manual_update: true
+manual_update: false
```

Comment:
**[Question, non-blocking]** Do we want to persist these changes to `inputs.yml`, or were these just for testing purposes?

Reply:
Ah, this was just for testing, I will revert.